Improve batch changes site admin guide (#25350)
sourcegraph/sourcegraph: doc/batch_changes/how-tos/site_admin_configuration.md

# Site admin configuration for Batch Changes

#### Setup Batch Changes 
1. Using Batch Changes requires a [code host connection](../../../admin/external_service/index.md) to a supported code host (currently GitHub, Bitbucket Server, and GitLab).

> 1. [Configure repository permissions](../../../admin/repo/permissions.md), which Batch Changes will respect.

Reviewer: ```suggestion
1. (Optional) [Configure repository permissions](../../../admin/repo/permissions.md), which Batch Changes will respect.
```

1. [Configure credentials](configuring_credentials.md).

> 1. Setup webhooks to make sure changesets sync fast. See [Batch Changes effect on codehost rate limits](../references/requirements.md#batch-changes-effect-on-code-host-rate-limits).

Reviewer 2: We might want to make it clearer that there are step by step instructions available:

```suggestion
1. Setup webhooks to make sure changesets sync fast. See [Batch Changes effect on codehost rate limits](../references/requirements.md#batch-changes-effect-on-code-host-rate-limits), and select your code host to get step-by-step instructions.
```

Or, alternatively, we could just link directly to the code host pages and save the user a click.

Author: Went with the alternative

> 1. (Optional) [Control the rate at which Batch Changes will create changesets](../../../admin/config/batch_changes.md#rollout-windows).

Reviewer: ```suggestion
1. (Optional) [Control the rate at which Batch Changes will publish changesets on code hosts](../../../admin/config/batch_changes.md#rollout-windows).
```
